inc/functions/cloudflare.php

```diff
@@ -41,15 +41,41 @@ function rocket_is_api_keys_valid_cloudflare( $cf_email, $cf_api_key, $cf_zone_i
 
 	try {
 		$cf_api_instance = new Cloudflare\Api( $cf_email, $cf_api_key );
-		$cf_user         = $cf_api_instance->get('user/');
-		$cf_zone         = $cf_api_instance->get('zones/'.$cf_zone_id);
+		$cf_user         = $cf_api_instance->get( 'user/' );
+		$cf_zone         = $cf_api_instance->get( 'zones/' . $cf_zone_id );
 
 		if ( ! isset( $cf_zone->success ) || empty( $cf_zone->success ) ) {
 			return new WP_Error( 'cloudflare_invalid_auth', $cf_zone->errors[0]->message );
 		}
 
 		if ( true === $cf_zone->success ) {
-		    return true;
+			$zone_found = false;
+
+			foreach ( $cf_zone->result as $zone ) {
```

Comment:
Is there a risk of the `result` property being undefined or empty? It might be good to check for that before using it in the loop